Assets/Mirror/Examples/Room/Scripts/Spawner.cs

```diff
@@ -6,12 +6,6 @@ public class Spawner : NetworkBehaviour
     {
         public NetworkIdentity prizePrefab;
 
-        float x;
```

Comment:
None of this has anything to do with separating RandomColor

Reply:
reverted